homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
         device_class=SensorDeviceClass.CURRENT,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "outputv": SensorEntityDescription(
         key="outputv",
-        name="UPS Output Voltage",
+        translation_key="output_voltage",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "reg1": SensorEntityDescription(
         key="reg1",
         translation_key="register_1_fault",
-        name="UPS Register 1 Fault",
         entity_registry_enabled_default=False,
     ),
     "reg2": SensorEntityDescription(
         key="reg2",
         translation_key="register_2_fault",
-        name="UPS Register 2 Fault",
         entity_registry_enabled_default=False,
     ),
     "reg3": SensorEntityDescription(
         key="reg3",
         translation_key="register_3_fault",
-        name="UPS Register 3 Fault",
         entity_registry_enabled_default=False,
     ),
     "retpct": SensorEntityDescription(
         key="retpct",
         translation_key="restore_capacity",
-        name="UPS Restore Requirement",
         native_unit_of_measurement=PERCENTAGE,
     ),
     "selftest": SensorEntityDescription(
         key="selftest",
         translation_key="self_test_result",
-        name="UPS Self Test result",
     ),
     "sense": SensorEntityDescription(
         key="sense",
         translation_key="sensitivity",
-        name="UPS Sensitivity",
         entity_registry_enabled_default=False,
     ),
     "serialno": SensorEntityDescription(
         key="serialno",
         translation_key="serial_number",
-        name="UPS Serial Number",
         entity_registry_enabled_default=False,
     ),
     "starttime": SensorEntityDescription(
         key="starttime",
         translation_key="startup_time",
-        name="UPS Startup Time",
     ),
     "statflag": SensorEntityDescription(
         key="statflag",
         translation_key="online_status",
-        name="UPS Status Flag",
         entity_registry_enabled_default=False,
     ),
     "status": SensorEntityDescription(
         key="status",
         translation_key="status",
-        name="UPS Status",
     ),
     "stesti": SensorEntityDescription(
         key="stesti",
         translation_key="self_test_interval",
-        name="UPS Self Test Interval",
     ),
     "timeleft": SensorEntityDescription(
         key="timeleft",
         translation_key="time_left",
-        name="UPS Time Left",
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "tonbatt": SensorEntityDescription(
         key="tonbatt",
         translation_key="time_on_battery",
-        name="UPS Time on Battery",
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
     "upsmode": SensorEntityDescription(
         key="upsmode",
         translation_key="ups_mode",
-        name="UPS Mode",
     ),
```

Comment:
enum candidate

Reply:
Yeah I agree, but I didn't find too much documentation on what possible values there are for this field (mine has `Stand Alone`).